deltachat-ios/AppDelegate.swift

```diff
@@ -125,16 +125,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         NotificationCenter.default.removeObserver(self, name: .reachabilityChanged, object: reachability)
     }
 
-    func dbfile() -> String {
```

Comment:
I would also use a computed property here as you did DatabaseHelper.

Reply:
This method has been deleted :).